application/src/main/kotlin/net/corda/v5/application/persistence/PersistenceService.kt

```diff
@@ -78,7 +79,8 @@ interface PersistenceService {
     fun <T : Any> find(entityClass: Class<T>, primaryKey: Any): T?
 
     /**
-     * Find multiple entities of the same type with different primary keys from the persistence context in a single transaction.
+     * Find multiple entities of the same type with different primary keys from the persistence context in a single
```

Comment:
`Find multiple entities of the same type with different primary keys from the persistence context in a single
     * transaction.` to `Find multiple entities of the same type with different primary keys in a single transaction`.